Src/BootCamp.Chapter/Gambling/Games/Poker/PokerCombo.cs

```diff
@@ -202,6 +214,56 @@ public static int CalculateHighCardScore(IEnumerable<Card> cards)
             return 0;
         }
 
+        public static int CalculateLowestCardScore(IEnumerable<Card> cards)
```

Comment:
Who's going to test what happens when 2 players have the same combos, but different score?